CHANGES.rst

```diff
@@ -300,6 +300,8 @@ New Features
   - Updated ``Gaussian2D`` to accept no arguments (will use default x/y_stddev
     and theta). [#5537]
 
+  - Added ``SmoothlyBrokenPowerLaw1D`` model.
```

Comment:
Move this entry to version 2.0.0 section. Version 1.3 is already released. Also, need to append PR number, see other entries as example.